### PR TITLE
Hide metrics from public 

### PR DESCRIPTION
**Pull request by linhkikuchi**
Related to https://github.com/brave/devops/issues/5867

Before: 
```
❯ http https://translate-relay.brave.com/metrics
HTTP/1.1 200 OK
Age: 9
Connection: keep-alive
Content-Encoding: gzip
Content-Length: 1475
Content-Type: text/plain; version=0.0.4; charset=utf-8
Date: Tue, 05 Oct 2021 14:48:53 GMT
Via: 1.1 d873eb6ebbb9da58c373c3c3b1843e77.cloudfront.net (CloudFront)
X-Amz-Cf-Id: _S8xWuclPl4NhpGam5B_6GPwiW1VUsaH60XJOnaPmgr7fh7nPDcb9g==
X-Amz-Cf-Pop: EWR52-C2
X-Cache: Hit from cloudfront
X-Edge-Origin-Shield-Skipped: 0
```

After
```
$ curl -I  https://translate-relay.brave.com/metrics
HTTP/2 404 
content-type: text/plain; charset=utf-8
content-length: 19
date: Wed, 06 Oct 2021 03:28:05 GMT
x-content-type-options: nosniff
x-cache: Error from cloudfront
via: 1.1 7140f0ca7678f315f05e94435ec1dea9.cloudfront.net (CloudFront)
x-amz-cf-pop: NRT57-P1
x-amz-cf-id: uIfaSIOi7KfyOyiiVKBrzhfLJhurTo_a5IsWIruHzZzIZvisZtuQDg==
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -54,8 +54,8 @@ RUN CGO_ENABLED=1 GOOS=linux go build \
 
 FROM ubuntu:20.04 as artifact
 
-RUN apt update
-RUN apt install -y git autoconf libtool g++ make
+RUN apt-get update
```
Comment (fmarier) [inline Dockerfile:57]: Not really a problem, but is that change needed? `apt` should be available on all distros.

Comment (wknapik) [reply Dockerfile:57]:
```console
% cat Dockerfile 
FROM ubuntu:20.04
RUN apt update
% hadolint Dockerfile 
Dockerfile:2 DL3027 warning: Do not use apt as it is meant to be a end-user tool, use apt-get or apt-cache instead
% docker build -t test .                     
Sending build context to Docker daemon  2.048kB
Step 1/2 : FROM ubuntu:20.04
 ---> 597ce1600cf4
Step 2/2 : RUN apt update
 ---> Running in f393fb47d390

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

[...]
%
```
